lib/fog/account/storm_on_demand.rb

```diff
@@ -1,8 +1,8 @@
 module Fog
   module Account
     class StormOnDemand < Fog::Service
-      autoload :Token, 'fog/account/storm_on_demand/models/token'
-      autoload :Tokens, 'fog/account/storm_on_demand/models/tokens'
+      autoload :Token, File.expand_path('../account/storm_on_demand/models/token', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.